### ISSUE TITLE
Old dependancies? any reason? 

### ISSUE THREAD

**Issue by shadowdogg** [AnomalyInnovations/serverless-stack-demo-ext-api#24]
```
  "devDependencies": {
    "aws-sdk": "^2.541.0",
    "serverless-bundle": "^1.2.5",
    "serverless-offline": "^5.12.0"
  },
```

Just wondering if there is any reason for this? As we are in 3.1.0 for serverless-bundle for example.

Assuming my understanding of the hats are correct

> It will update you to the most recent major version (the first number). ^1.2.3 will match any 1.x.xrelease including 1.3.0, but will hold off on 2.0.0.

Cheers.

**Comment by jayair** [comment]
Yeah we haven't updated the versions recently. Feel free to update them and submit a PR.
